lib/mix/phoenix.ex

```diff
@@ -93,6 +93,16 @@ defmodule Mix.Phoenix do
   end
 
   @doc """
+  Checks the availability of a given module name.
+  """
+  def check_module_name_availability!(name) do
```

Comment:
I copied the `check_module_name_availability!` verbatim, as I saw little need to modify it.